test/unit/abilities/end_users_test.rb

```diff
@@ -45,4 +46,48 @@ def setup
     assert_can ability, :see, :end_users
     assert_cannot ability, :manage, :end_users
   end
+
+  class Onpremises < ActiveSupport::TestCase
+    def setup
+      @provider = FactoryBot.create(:provider_account)
```

Comment:
Maybe this is enough:
```suggestion
      @provider = FactoryBot.create(:simple_provider)
```

Reply:
Well spotted, but not worth it. This whole end-users thing will vanish pretty soon.